L/libwebp/build_tarballs.jl

```diff
@@ -0,0 +1,70 @@
+# Note that this script can accept some limited command-line arguments, run
+# `julia build_tarballs.jl --help` to see a usage message.
+using BinaryBuilder
+
+name = "libwebp"
+version = v"1.0.3"
+
+# Collection of sources required to build libwebp
+sources = [
+    "https://storage.googleapis.com/downloads.webmproject.org/releases/webp/libwebp-$(version).tar.gz" =>
+    "e20a07865c8697bba00aebccc6f54912d6bc333bb4d604e6b07491c1a226b34f",
+    "./bundled",
+]
+
+# Bash recipe for building across all platforms
+script = raw"""
+cd $WORKSPACE/srcdir/libwebp-*/
+export CFLAGS="-std=c99"
+export CPPFLAGS="-I${prefix}/include"
+if [[ "${target}" == *-apple-* ]] || [[ "${target}" == *-freebsd* ]]; then
+    # Clang emits a warning about unused command line options.  configure runs
+    # several tests forcing `CFLAGS` to have `-Werror`, without the possibility
+    # to override this setting from the outside, but this causes a lot of tests
+    # to fail because of the pesky Clang's warning.  The following patch adds
+    # `-Wno-unused-command-line-options` next to `-Werror`.
+    atomic_patch -p1 ../patches/configure_ac_Wno-unused-command-line-options.patch
+    autoreconf
+fi
+if [[ "${target}" == *-freebsd* ]]; then
+    export LDFLAGS="-L${libdir}"
```

Comment:
I drained all my energy to understand what was the problem with the failing tests, so I couldn't figure out _why_ on FreeBSD I needed to explicitly set `LDFLAGS` this way